develop/getting-started/using-the-ide.md

```diff
@@ -0,0 +1,246 @@
+---
+title: IDE Tips and Tricks
+description: Useful information to handle and traverse your Project using the IDE efficiently.
+authors: 
+    - JR1811
+---
+
+# IDE Tips and Tricks
+
+This page gives useful bits of information, to speed up and ease the workflow of developers. Incorporate them into yours, to your liking.
+It may take some time to learn and get used to the shortcuts and other options. You can use this page as a reference for that.
+
+> [!NOTE]
+> Key binds in the text are listed as Windows shortcuts.
+> Refer to the `File > Setings > Keymap` Settings or search for the functionality elsewhere if you are using a different Keyboard layout.
+
+## Traversing projects
+
+### Manually
+
+IntelliJ has many different ways of traversing projects. If you have generated sources using the `./gradlew genSources` commands in the terminal or used the `Tasks > fabric > genSources` Gradle Tasks in the Gradle Window, you can manually go through the source files of Minecraft in the
+Project Window's External Libraries.
+
+![Gradle Task](/assets/develop/getting-started/using-the-ide/traversing_01.png)
+
+::: details Recommendation
+A recommendation would be to aim for a Java decompiler, which is as accurate as possible. For example, the [VineFlower](https://github.com/Vineflower/vineflower) option allows for a good experience when debugging the source code of Minecraft.
+
+If you have issues with one of the options, you can always switch later on too, simply by running their Gradle Tasks.
+:::
+
+The Minecraft Source code can be found if you look for `net.minecraft` in the Project Window's External Libraries. If you use the "Split client and common sources" option from the online [Template mod generator](https://fabricmc.net/develop/template/), there will be two sources, as the name suggests. But also other sources of projects, libraries and dependencies, which are imported using the `build.gradle` file
+are located in there. This method is often used when browsing for assets, tags and other files.
+
+![External Library](/assets/develop/getting-started/using-the-ide/traversing_02_1.png)
+
+![Split sources](/assets/develop/getting-started/using-the-ide/traversing_02_2.png)
+
+### Search
+
+Pressing `Shift` twice opens up a Search window. In there you can search for your project's files and classes. When Activating the checkbox `include non-project items`
```

Comment:
You should put a notice at the top of the page about keybinds being the default keymap, some people switch their IntelliJ IDEA keymaps when they first install it without batting a thought.

Also, it is probably worth using the `<kbd>` tag here for keybinds instead of inline codeblocks?

Reply:
Well, not all of them are the default ones. E.g. in the comments section, it's explained how to set up custom key binds for that. But yeah, a note for that should be good tho.